proc/api.thrift

```diff
@@ -0,0 +1,40 @@
+service Proc {
+  
+  exception InvalidRequest {
+    1: optional string message
+  }
+
+  exception ScootTooBusy {
+    1: optional i64 retryAfterMs
+  }
+  
+  enum JobType {
+    IRON_TESTS=1,
+    DEFAULT=2,
```

Comment:
So I set it to 1 because sometimes its difficult to tell between no value, (defaults to 0) and specifying default (also 0) since this is optional.  

Maybe it doesn't really matter in this case but typically I don't start IDL enums at 0